packages/packagers/js/src/helpers.js

```diff
@@ -13,7 +13,7 @@ if (parcelRequire == null) {
     if (id in $parcel$inits) {
       let init = $parcel$inits[id];
       delete $parcel$inits[id];
-      let module = {id, exports: {}};
+      let module = {id: id, exports: {}};
```

Comment:
isn't the `let` declaration itself also a problem?

Reply:
It seems IE11 supports some variants of `let`: https://caniuse.com/let